### PR TITLE
Update environment dependencies in pubspec

### PR DESCRIPTION
**Pull request by cacheung**
Update environment dependencies in pubspec.
Update versions to 4.0.2 for all plugin except aepedge is 4.1.0. 
Update Readme for Edge Bridge.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: plugins/flutter_aepedgebridge/ios/flutter_aepedgeBridge.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'flutter_aepedgebridge'
-  s.version          = '4.0.0'
+  s.version          = '4.0.2'
```
Comment (cacheung) [inline plugins/flutter_aepedgebridge/ios/flutter_aepedgeBridge.podspec:3]: podspec update from 4.0.0 to 4.0.2 due to I miss to update the 4.0.1 version

File: plugins/flutter_aepedge/ios/flutter_aepedge.podspec [hunk 2]
```diff
@@ -3,7 +3,7 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'flutter_aepedge'
-  s.version          = '4.0.0'
+  s.version          = '4.1.0'
```
Comment (cacheung) [inline plugins/flutter_aepedge/ios/flutter_aepedge.podspec:6]: Edge is version 4.1.0, we added datastreamOverride feature.

File: plugins/flutter_aepassurance/ios/flutter_aepassurance.podspec [hunk 3]
```diff
@@ -3,7 +3,7 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'flutter_aepassurance'
-  s.version          = '4.0.0'
+  s.version          = '4.0.2'
```
Comment (addb) [inline plugins/flutter_aepassurance/ios/flutter_aepassurance.podspec:6]: why are we jumping from 0 to 2?

Comment (cacheung) [reply plugins/flutter_aepassurance/ios/flutter_aepassurance.podspec:6]: I missed to update that for 4.0.1.  It should be matching to the aep flutter version, so correct to 4.0.2 in this PR.

File: plugins/flutter_aepedgebridge/ios/flutter_aepedgeBridge.podspec (hunk 1 above)
Comment (addb) [reply plugins/flutter_aepedgebridge/ios/flutter_aepedgeBridge.podspec:3]: Will this be an issue? Or it is fine if they are not able to access 4.0.1 version?

File: plugins/flutter_aepedge/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+## 4.1.0
+
+* Added support to customize datastream configurations on a per-event basis using the sendEvent API
+* Update enviornment dependencies in pubspec.
```
Comment (praveek) [inline plugins/flutter_aepedge/CHANGELOG.md:4]: Typo: environment in all the CHANGELOG.md file

File: plugins/flutter_aepedgebridge/ios/flutter_aepedgeBridge.podspec (hunk 1 above)
Comment (cacheung) [reply plugins/flutter_aepedgebridge/ios/flutter_aepedgeBridge.podspec:3]: This version shows up when doing the pod update.

e.g flutter_aepcore (4.0.2)

So should match to Pubspec version.  It doesn't affect functionality.

podupdate

**Review by praveek — APPROVED** [review]

**Review by addb — APPROVED** [review]

**Pull request merged**